### ISSUE TITLE
have to manually install @types/ledgerhq__hw-transport

### ISSUE THREAD

**Issue by ExtraHash** [turtlecoin/turtlecoin-wallet-backend-js#102]
when i install this module in a new typescript project, import it and run "yarn build" i get errors saying that type definitions for @ledgerhq/hw-transport cannot be found. i had to manually run

```
yarn @types/ledgerhq__hw-transport
```

and then it worked.

**Comment by zpalmtree** [comment]
try now

**Comment by ExtraHash** [comment]
still getting the same error trace on yarn build

```
yarn run v1.22.4
$ node ./node_modules/typescript/bin/tsc
node_modules/turtlecoin-utils/dist/LedgerDevice.d.ts:1:23 - error TS2688: Cannot find type definition file for 'ledgerhq__hw-transport'.

1 /// <reference types="ledgerhq__hw-transport" />
                        ~~~~~~~~~~~~~~~~~~~~~~

node_modules/turtlecoin-utils/dist/LedgerDevice.d.ts:3:23 - error TS7016: Could not find a declaration file for module '@ledgerhq/hw-transport'. '/home/ender/code/web-wallet/node_modules/@ledgerhq/hw-transport/lib/Transport.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/ledgerhq__hw-transport` if it exists or add a new declaration (.d.ts) file containing `declare module '@ledgerhq/hw-transport';`

3 import Transport from '@ledgerhq/hw-transport';
                        ~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/turtlecoin-utils/dist/LedgerNote.d.ts:1:23 - error TS2688: Cannot find type definition file for 'ledgerhq__hw-transport'.

1 /// <reference types="ledgerhq__hw-transport" />
                        ~~~~~~~~~~~~~~~~~~~~~~

node_modules/turtlecoin-utils/dist/LedgerNote.d.ts:5:23 - error TS7016: Could not find a declaration file for module '@ledgerhq/hw-transport'. '/home/ender/code/web-wallet/node_modules/@ledgerhq/hw-transport/lib/Transport.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/ledgerhq__hw-transport` if it exists or add a new declaration (.d.ts) file containing `declare module '@ledgerhq/hw-transport';`

5 import Transport from '@ledgerhq/hw-transport';
                        ~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/turtlecoin-wallet-backend/dist/lib/Config.d.ts:1:23 - error TS2688: Cannot find type definition file for 'ledgerhq__hw-transport'.

1 /// <reference types="ledgerhq__hw-transport" />
                        ~~~~~~~~~~~~~~~~~~~~~~


Found 5 errors.

```

**Comment by ExtraHash** [comment]
looks like it's from @brandonlehmann 's library

**Comment by brandonlehmann** [comment]
Weird... is this not it? https://github.com/turtlecoin/turtlecoin-utils/blob/0a2ef011bb8d4cc44f16c0be410a0b64a5604e44/package.json#L62

**Comment by brandonlehmann** [comment]
> Weird... is this not it? https://github.com/turtlecoin/turtlecoin-utils/blob/0a2ef011bb8d4cc44f16c0be410a0b64a5604e44/package.json#L62

Ah, strike that, it's a dev dependency so it's not loaded as part of the install from NPM.

**Comment by brandonlehmann** [comment]
> > Weird... is this not it? https://github.com/turtlecoin/turtlecoin-utils/blob/0a2ef011bb8d4cc44f16c0be410a0b64a5604e44/package.json#L62
> 
> Ah, strike that, it's a dev dependency so it's not loaded as part of the install from NPM.

Just moved all the third-party @types into dependencies in https://github.com/turtlecoin/turtlecoin-utils/commit/13feb30d50aee81058769a8018fd735a35cac5fa and released as 3.0.9 of utils.

**Comment by ExtraHash** [comment]
appears to be fixed i believe
